README.md

```diff
@@ -24,11 +24,13 @@ use Pixels\Components\SocialShare\Share;
 $facebook_share = Share::facebook();
 $twitter_share  = Share::twitter();
 $linkedin_share = Share::linkedin();
+$whatsup_share  = Share:: whatsup();
```

Comment:
Took me a while to realize why it looked funny. It's supposed to be "whatsapp", not "whatsup"

Reply:
Better to search/replace it to all files